core/src/test/java/org/testcontainers/junit/GenericContainerRuleTest.java

```diff
@@ -121,6 +123,13 @@ public static void setupContent() throws FileNotFoundException {
             .withExtraHost("somehost", "192.168.1.10")
             .withCommand("/bin/sh", "-c", "while true; do cat /etc/hosts | nc -l -p 80; done");
 
+    /**
+     * Create a container which wait for some time to test if copy to container works.
+     */
+    @ClassRule
+    public static GenericContainer alpineCopyToContainer = new GenericContainer("alpine:3.2")
```

Comment:
Actually, it would be nice if you can replace it with `try-with-resources` version (see `NetworkTest.WithoutRules`). It helps to run individual tests without starting everything